checkov/arm/checks/resource/ACRGeoreplicated.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+from typing import Any
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.arm.base_resource_check import BaseResourceCheck
+
+
+class ACRGeoreplicated(BaseResourceCheck):
+    def __init__(self) -> None:
+
+        name = "Ensure geo-replicated container registries to match multi-region container deployments."
+        id = "CKV_AZURE_165"
+        supported_resources = ("Microsoft.ContainerRegistry/registries",)
```

Comment:
Looks like u should scan the `Microsoft.ContainerRegistry/registries/replications` resource and look for the `location` field.